src/test/java/it/gov/pagopa/payment/controller/payment/BarCodePaymentControllerIntegrationTest.java

```diff
@@ -87,7 +93,8 @@ void createTransaction_initiativeNotExistent() throws Exception {
         extractResponse(createTrx(trxRequest, USERID), HttpStatus.NOT_FOUND, null);
 
         AuthBarCodePaymentDTO authPaymentDTO = AuthBarCodePaymentDTO.builder().amountCents(10000L).build();
-        extractResponse(authTrx(trxCode, authPaymentDTO, MERCHANTID), HttpStatus.NOT_FOUND, null);
+        Object response = extractResponse(authTrx(trxCode, authPaymentDTO, MERCHANTID), HttpStatus.NOT_FOUND, null);
+        assertNull(response);
```

Comment:
this assert null is useless! the extractResponse test method will always return null if "null" is the expected configured class! thus you are testint the extractResponse method this way! we are not interested on it!
If you want to check if there is not body, you should instead modify the extractResponse itself to check the empty body!